pmm-app/src/pmm-add-instance/components/AddInstance/AddInstance.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react';
+import { useTheme } from '@grafana/ui';
+import { Database } from 'shared/components/Elements/Icons/Database';
+import { getStyles } from './AddInstance.styles';
+import { instanceList } from './AddInstance.constants';
+import { Messages } from './AddInstance.messages';
+import { AddInstanceProps, SelectInstanceProps } from './AddInstance.types';
+
+export const SelectInstance: FC<SelectInstanceProps> = (props) => {
```

Comment:
Why not destructure the props here since?

Reply:
done